### ISSUE TITLE
Fix test in test-amp-ad-xorigin-iframe-handler

### ISSUE THREAD

**Issue by samouri** [ampproject/amphtml#30621]
**summary**
While addressing https://github.com/ampproject/amphtml/pull/30578, an unrelated test failure became blocking. The failure was pre-existing within _master_, so I opted to `skip` it. This issue is for attempting to debug the issue and figure out why the test is failing / fix it.

Test: `should be able to use user-error API` within extensions/amp-ad/0.1/test/test-amp-ad-xorigin-iframe-handler.js.
